### PR TITLE
Som ERT breacher skills

### PR DESCRIPTION
**Pull request by Lumipharon**
## About The Pull Request
Adds some skills to the SOM breacher ert/the SOM SL, including the axe sweep ability.
Actually gives them their own name so its not just a copy paste.
## Why It's Good For The Game
Big more interesting variety, makes the breacher variant a little better considering they have a strong focus on melee with axes.
## Changelog
:cl:
balance: SOM breacher ERTs have undergone more advanced training
/:cl:

### CODE REVIEW

File: code/datums/jobs/job/sons_of_mars_ert.dm [hunk 1]
```diff
@@ -342,6 +343,8 @@
 
 /datum/outfit/job/som/ert/veteran/breacher_vet/post_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
 	. = ..()
+	H.add_traits(list(TRAIT_AXE_EXPERT, TRAIT_SWORD_EXPERT), TRAIT_GENERIC)
```
Comment (TiviPlus) [inline code/datums/jobs/job/sons_of_mars_ert.dm:346]: shouldnt all these traits be trait inate?
And shouldnt they be on the job?

**Review by TiviPlus — CHANGES_REQUESTED** [review]

File: code/datums/jobs/job/sons_of_mars_ert.dm (hunk 1 above)
Comment (Lumipharon) [reply code/datums/jobs/job/sons_of_mars_ert.dm:346]: I'll update it, whats the diff tho? Also they are on the job?

Comment (TiviPlus) [reply code/datums/jobs/job/sons_of_mars_ert.dm]: No they're on the outfit
last i checked we apply skills based on job datum

Comment (Lumipharon) [reply code/datums/jobs/job/sons_of_mars_ert.dm]: Oh whoops, updated

**Pull request merged**